skimage/graph/tests/test_pixel_graph.py

```diff
@@ -46,3 +46,12 @@ def test_default_edge_func():
     assert num_edges == 12  # lattice in a (3, 3) grid
     np.testing.assert_almost_equal(g[0, 1], 0.78 * np.abs(image[0, 0] - image[0, 1]))
     np.testing.assert_array_equal(n, np.arange(image.size))
+
+
+def test_no_mask_with_edge_func():
```

Comment:
Maybe add a docstring to clarify what we're testing?
```suggestion
def test_no_mask_with_edge_func():
    """Ensure function `pixel_graph` runs when passing `edge_function` but not `mask`."""
```